README.md

```diff
@@ -300,6 +262,47 @@ or shell option args
 
 ----
 
+
+##### `gcr.io/onec-co/datamon-fuse-sidecar` -- `wrap_datamon.sh`
+
+Provides filesystem representations (i.e. a folder) of
+[datamon bundles](#data-modeling).
+The environment variables consist of
+* `dm_fuse_opts` global options
+* `dm_fuse_bd_<id>` bundle options
+
+The available global options are
+
+* `c` the same coordination directory passed to `wrap_application.sh`
+* `n`, `e` name and email for datamon contributor record
+* `S` without an `<arg>` is causes the wrapper script to sleep instead
+  of exiting, which can be useful for debug.
+
+The per-bundle options (one per `<id>`,
+a unique string among all bundle options) determine whether the bundle
+is a _source_, data that exist in datamon and accessed by sidecar,
+or a _destination_, data that are created by the sidecar.
```

Comment:
> created by the sidecar
Aren't they created by the main container, not the sidecar?

Reply:
> created by the main container and stored in datamon by the sidecar

is a more full explain

> created by the main container

is also a possible re-phrase...
... erring on the side of terseness with the readme, overall.

could change to the more verbose explain or `s/sidecar/main container/`.  no particularly strong opines here.